compiler+runtime/src/cpp/jank/read/lex.cpp

```diff
@@ -542,26 +681,79 @@ namespace jank::read
             }
 
             /* Tokens beginning with - are ambiguous; it's only a negative number if it has numbers
-             * to follow.
-             * TODO: handle numbers starting with `+` */
+                     * to follow.
+                     * TODO: handle numbers starting with `+` */
```

Comment:
Looks like an incorrect change.

Reply:
We can remove that TODO now, right? We handle numbers starting in `+`. Hmm, I don't see any tests for them though.

Reply:
right, i didn't do anything regarding '+', so i'll keep it here.